api-gateway/caching-api-response/modules/example-action/src/main/java/com/project/example/action/ExampleActionFactory.java

```diff
@@ -0,0 +1,30 @@
+package com.project.example.action;
+
+import io.knotx.fragments.api.Fragment;
+import io.knotx.fragments.handler.api.Action;
+import io.knotx.fragments.handler.api.ActionFactory;
+import io.knotx.fragments.handler.api.domain.FragmentResult;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+
+public class ExampleActionFactory implements ActionFactory {
```

Comment:
I think we can remove all modules from starter kit. We do not use them (despite of `healthcheck`).

Reply:
fixed